adapter/integration-test/src/adapter-datasource.test.js

```diff
@@ -184,6 +187,52 @@ describe('Adapter Configuration', () => {
   })
 })
 
+test('POST datasources/{id}/trigger', async () => {
+  const dynamicDatasourceResponse = await request(URL)
+    .post('/datasources')
+    .send(dynamicDatasourceConfig)
+  const dynDatasourceId = dynamicDatasourceResponse.body.id
+
+  const dynamicDataMetaData = await request(URL)
+    .post('/datasources/' + dynDatasourceId + '/trigger')
```

Comment:
Little pointer to how you can concat strings very nicely in JavaScript:
```
`/datasources/${dynDatasourceId}/trigger`
```

Reply:
Thanks for the suggestion. I changed it right away.